### DISCUSSION TITLE
Queue.close() required?

### DISCUSSION THREAD

**Discussion by Vendrad** [OptimalBits/bull#2071] [Q&A]
Taking the following simple example. The relevant entries were being added to Redis but the process itself wasn't completing until I manually added `testQueue.close();`. Other than the reference document letting me know this function exists and a bit of trial and error no examples actually stated that you should close the queue after adding an item.

```javascript
import Queue from "bull";

const add = async () => {
  const testQueue = new Queue("tasks");
  await testQueue.add({ difficulty: "simple" });

  // Required...?
  testQueue.close();
};

(async () => {
  await add();
  console.log("job added");
})();
```

Is this implementation correct or have I missed something else? If it is correct then maybe consider this as a suggestion to update the documentation.

**Comment by manast** [comment]
Definitely not needed to close in order to add jobs to the queue... something else going on.
